File: Getting Started/java/PrintFibonacciTillN.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        // write your code here
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();
        int f1 = 0;
        int f2 = 1;
        System.out.println(f1);
        System.out.println(f2);
        for (int i = 2; i < n; i++) {
            int f3 = f1 + f2;
            System.out.println(f3);
            f1 = f2;
            f2 = f3;

        }
    }
}

File: 1. Getting Started/cpp/PrintFibonacciTillN.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
    int n;
    cin >> n;
    int arr[n];
    arr[0]=0;
    arr[1]=1;
    for(int i=2;i<n;i++){
        arr[i]=arr[i-1]+arr[i-2];
    }
    for(int i=0;i<n;i++){
        cout << arr[i] << endl;
    }
    
    //write your code here
    
}